examples/providers/README.md

```diff
@@ -0,0 +1,20 @@
+# Providers Example
```

Comment:
Is this really a provider example? Or is it a web scraping/firecrawl example that happens to show you how to create a provider? Should we just call this "Web scraping example"?

Reply:
hmm interesting question. I think this is really a question on how users will use and look through examples and how we can optimize for that. 

Users could either:
a. Be looking for an example for a scenario to help implement something
b. Be looking to learn how to implement some pattern (structured outputs, providers)

My hypothesis is that b is more common and more useful for simple examples like this. Or maybe a different spin is that we want some examples that show a and others that show b

Web scraping alone is a good scenario to show although there's not much meat here on that front besides saying "you can use firecrawl"

Reply:
I'm fine with that.